nuxeo-runtime/nuxeo-launcher-commons/src/main/java/org/nuxeo/launcher/config/ConfigurationGenerator.java

```diff
@@ -1394,6 +1394,12 @@ public InetAddress getBindAddress() throws ConfigurationException {
         InetAddress bindAddress;
         try {
             bindAddress = InetAddress.getByName(userConfig.getProperty(PARAM_BIND_ADDRESS));
+            if (bindAddress.isAnyLocalAddress()) {
+                boolean preferIPv6 = "false".equals(System.getProperty("java.net.preferIPv4Stack"))
```

Comment:
No, these are Java system properties controlling the Java runtime, we shouldn't go through the Nuxeo Framework mechanism for that.
Note that there exists `Boolean.getProperty` to parse a system property as a boolean, but it can return `false` in more cases, whereas here the code tests for an explicit `"false"` string.
As long as this work I say keep it as is, it's not crucial code.